src/rgw/rgw_crypt.cc

```diff
@@ -35,8 +35,6 @@ using namespace rgw;
 /**
  * Encryption in CTR mode. offset is used as IV for each block.
  */
-#warning "TODO: move this code to auth/Crypto for others to reuse."
```

Comment:
maybe @aclamk remembers why the #warnings, I vaguely remember it might have been @liewegas that asked to do it. Anyhow, it'd be better to revert 326a16da26dd3d59a7875580343a7da917905201.